conjur-oss/templates/deployment.yaml

```diff
@@ -131,6 +131,13 @@ spec:
             secretKeyRef:
               name: {{ .Release.Name }}-conjur-database-url
               key: key
+        {{ if .Values.database.remotePassword }}
+        - name: PGPASSWORD
```

Comment:
Is `PGPASSWORD` a new environment variable for the Conjur image? How will this be used by Conjur?